src/msg.rs

```diff
@@ -237,15 +243,15 @@ pub enum QueryMsg {
         page: Option<u32>,
         page_size: u32,
     },
-    Test {},
+    Minters {},
 }
 
 impl QueryMsg {
     pub fn get_validation_params(&self) -> (&HumanAddr, ViewingKey) {
         match self {
             Self::Balance { address, key } => (address, ViewingKey(key.clone())),
             Self::TransferHistory { address, key, .. } => (address, ViewingKey(key.clone())),
-            _ => (panic!("lol")),
+            _ => panic!("Unimplemented authenticated query type"),
```

Comment:
Changed to `"Unknown Query type"`. Makes more sense to me, since the user doesn't necessarily intend to reach the authenticated queries part of the code, he may just write the query message wrong.

Reply:
The user will never see this error as long as the contract is implemented correctly, so i'm thinking it should be an error that is informative to someone looking at test logs

Reply:
This looks better to me now, take a look (anything other than specified authenticated queries won't reach there)